### PR TITLE
@xtina-starr => Callouts inline

### PR DESCRIPTION
**Pull request by eessex**
closes https://github.com/artsy/force/issues/260
Updated styling for callout articles and pull-quotes.

### CODE REVIEW

**Comment by eessex** [comment]
screen shot 2016-10-10 at 5 24 32 pm
screen shot 2016-10-10 at 5 30 20 pm

File: components/article/stylesheets/index.styl [hunk 1]
```diff
@@ -445,6 +445,7 @@ body.body-article
       margin-right 20px
     .article-section-callout
       garamond-size('l-body')
+      width "calc(100% - %s)" % (140px)
```
Comment (xtina-starr) [inline components/article/stylesheets/index.styl]: Nice use of calc! 👌 

Noticed you are using string interpolation here with a fixed value of `140px` versus just passing the px value in directly. I don't use calc often so I may just be unfamiliar, but would `calc(100% - 140px)` work? Like I said that's really minor and not enough to hold up the PR 😄

**Comment by xtina-starr** [comment]
⭐️

File: components/article/stylesheets/index.styl (hunk 1 above)
Comment (eessex) [inline components/article/stylesheets/index.styl]: Tried this but stylus doesn't seem to play nice with calc if the units are different

**Pull request merged**